### ISSUE TITLE
feautre/create-monster

### ISSUE THREAD

**Issue by bmkersey** [BenHuf/encounter-gen#9]
create a custom creature function

use a modal to collect all info

populate on screen

**Comment by BenHuf** [comment]
Instead of using a modal this can just append an empty template card to the page.

**Comment by bmkersey** [comment]
yeah i figured that would be an easy one for you. I went ahead and assigned it to you.

**Comment by BenHuf** [comment]
Pushing this to the back burner until after MVP is finished.
